salt/states/ceph.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import json
+from salt.exceptions import CommandExecutionError, CommandNotFoundError
+
+try:
+    import ceph_cfg
+    HAS_CEPH_CFG = True
+except ImportError:
+    HAS_CEPH_CFG = False
+
+log = logging.getLogger(__name__)
+
+
+def _unchanged(name, msg):
+    return {'name': name, 'result': True, 'comment': msg, 'changes': {}}
+
+
+def _test(name, msg):
+    return {'name': name, 'result': None, 'comment': msg, 'changes': {}}
+
+
+def _error(name, msg):
+    return {'name': name, 'result': False, 'comment': msg, 'changes': {}}
+
+
+def _changed(name, msg, **changes):
+    return {'name': name, 'result': True, 'comment': msg, 'changes': changes}
+
+
+def _ordereddict2dict(input_ordered_dict):
+    return json.loads(json.dumps(input_ordered_dict))
+
+
+def quorum(name, **kwargs):
+    """
+    Quorum state
+
+    This state is needed to allow the cluster to function.
+
+    Example usage in sls file:
```

Comment:
Can you add a `.. code-block:: yaml` directive here, and then adjust the spacing for the example yaml file below?